SoftLayer/CLI/firewall/detail.py

```diff
@@ -12,31 +12,70 @@
 
 @click.command(cls=SoftLayer.CLI.command.SLCommand, )
 @click.argument('identifier')
+@click.option('--credentials', type=click.BOOL,
+              help="Display FortiGate username and FortiGate password to multi vlans.")
 @environment.pass_env
-def cli(env, identifier):
+def cli(env, identifier, credentials):
     """Detail firewall."""
```

Comment:
I think this command would benefit from an example in this docblock.

Reply:
Two examples about vs and multi vlan were added in dockblock